tracer/src/Datadog.Trace/Configuration/IntegrationId.cs

```diff
@@ -63,5 +63,6 @@ internal enum IntegrationId
         Ssrf,
         AwsKinesis,
         AzureServiceBus,
+        Random,
```

Comment:
Given this is a such a generic name, I wonder whether we should call it `SystemRandom`, or `WeakRandom` or something to avoid any future clashes 🤔

Reply:
Sure!